tianshou/policy/imitation/cql.py

```diff
@@ -45,6 +45,7 @@ class CQLPolicy(SACPolicy):
     :param float alpha_min: lower bound for clipping cql_alpha. Default to 0.0.
     :param float alpha_max: upper bound for clipping cql_alpha. Default to 1e6.
     :param float clip_grad: clip_grad for updating critic network. Default to 1.0.
+    :param calibrated: calibrate Q-values as in CalQL paper arXiv:2303.05479.
```

Comment:
Just referencing the paper is not enough, pls provide a short explanation of what this does and when it makes sense to use this

Reply:
Added information about use case